packages/web-components/src/accordion/accordion-item/accordion-item.styles.ts

```diff
@@ -85,6 +85,7 @@ export const AccordionItemStyles = css`
         grid-column: 4;
         z-index: 2;
         pointer-events: none;
+        fill: var(--accent-fill-rest);
```

Comment:
I think this should use `accentFillRestBehavior.var`

Reply:
Good catch @nicholasrice

Reply:
thanks @nicholasrice . updated